### DISCUSSION TITLE
Simple moving average

### DISCUSSION THREAD

**Discussion by Zourvan** [opensource9ja/danfojs#480] [Q&A]
Hi,
Hi, I have simple question, I tried to rewrite Pandas_TA in danfojs. so may I have simple question about rolling calculation, Is any internal way to for example calculate simple moving average???

**Comment by steveoni** [comment]
@Zourvan that's a great idea..

we currently don't have that implemented... But it is possible to implement that in Danfojs..

let me know if there is any way I could help

**Reply by skedee** [reply]
SMA and EMA built into danfos I suspect would help those in the FinTech space trying to move away from Pandas.    In the meantime, I've been looking at https://github.com/kaelzhang/finmath to help with SMA and EMA.
